backend/src/models/community.py

```diff
@@ -0,0 +1,10 @@
+import dataclasses
+
+from backend.src.models.participant import Participant
```

Comment:
maybe not split it in a 100 files because the importing becomes complicated and you have to open many tabs